component/build_info.go

```diff
@@ -28,6 +28,9 @@ type BuildInfo struct {
 }
 
 // DefaultBuildInfo returns the default BuildInfo.
+//   Command: otelcol
```

Comment:
A developer can navigate to the source to read this. I don't think we should document these values in the godoc.

Reply:
Updated this change.